mailviews/previews.py

```diff
@@ -27,7 +22,12 @@
 from django.utils.module_loading import module_has_submodule
 
 from mailviews.helpers import should_use_staticfiles
-from mailviews.utils import split_docstring, unimplemented
+from mailviews.utils import split_docstring, unimplemented, is_django_version_greater
+
+if is_django_version_greater('1.9'):
+    from django.conf.urls import include, url
+else:
+    from django.conf.urls import patterns, include, url
```

Comment:
It looks like `patterns` is no longer being used here, so you should be able to drop the conditional and just import `include` and `url`.